### PR TITLE
release: 2.61

### PR DESCRIPTION
**Pull request by Meulengracht**

### CODE REVIEW

File: NEWS.md [hunk 1]
```diff
@@ -1,3 +1,22 @@
...
+* interfaces: add pcscd interface
+* Kernel command-line can now be edited in the gadget.yaml
+* Only track validation-sets in run-mode, fixes validation-set issues on first boot.
+* Added support for using store.access to disable access to snap store
+* Support for fat16 partition in gadget
+* Pre-seed authority delegation is now possible
+* Support new system-user name  _daemon_
+* Several bug fixes and improvements around remodelling
+* Offline remodelling support
+* Offline device mode support
```
Comment (pedronis) [inline NEWS.md:18]: what is this last line about?

is this the same as "Added support for using store.access to disable access to snap store" above?

**Review by pedronis — COMMENTED** [review]
question

File: NEWS.md (hunk 1 above)
Comment (pedronis) [reply NEWS.md:18]: is it about device-service.access=offline? in which case it is too cryptic I think

Comment (Meulengracht) [reply NEWS.md:18]: Good catch, can see we went a bit overboard with the NEWS added. I'll get that fixed

Comment (Meulengracht) [reply NEWS.md]: It was not about the device-service.access (i believe I see different entries for that one in the commits), the last one was added from memory, and the first one was added from the changelog we generated, so we should have removed the last one

**Review by pedronis — APPROVED** [review]
lgtm

**Pull request merged**